quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/Environment.java

```diff
@@ -63,6 +64,10 @@ public static String getHomeDir() {
         return System.getProperty("kc.home.dir");
     }
 
+    public static Boolean isProduct() {
+        return Boolean.getBoolean(PRODUCT);
+    }
```

Comment:
Me either. I'm not sure what exactly you mean by "handle the logic purely in maven" because this should solve the problem with exposed GELF configuration options. We need to hide it and accordingly asses tests. However, checking the classpath might be sufficient and easier to implement.

Reply:
By "handle the logic purely in maven" I mean that Keycloak won't care about if something's a product, or not. All will be based on if GELF is present, and this logic (whether GELF is present) will be handled purely by maven.